generator/src/main/java/org/corfudb/generator/distributions/Streams.java

```diff
@@ -1,5 +1,7 @@
 package org.corfudb.generator.distributions;
 
+import org.corfudb.runtime.CorfuRuntime;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.runtime.CorfuRuntime'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)